src/Service/PagePurger.php

```diff
@@ -43,6 +44,24 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.x
+	 *
+	 * @param Pages $pages
+	 *
+	 * @return bool
+	 *
+	 */
+	public function purgePages( Pages $pages ) {
+		$pagesArray = $pages->toArray();
+
+		foreach ( $pagesArray as $page ) {
+			$this->purge( $page );
+		}
+
+		return true;
```

Comment:
I can return true/false if the request have succeed `$this->api->postRequest(..)` returns an array and I can check for the response. It is a good idea to apply this logic at `purge()` too.

Another idea is to return the response array, this idea gives more freedom but those who will use the library will have to do more check (if needed).